src/auxiliary/auxiliary.jl

```diff
@@ -345,4 +345,11 @@ function register_error_hints()
 
     return nothing
 end
+
+get_backend(::PtrArray) = CPU()
+
+function get_array_type(backend::CPU)
+    return Array
+end
```

Comment:
Those should inline anyways, but this might give the compiler even more motivation to do so
```suggestion
@inline get_backend(::PtrArray) = CPU()

@inline get_array_type(backend::CPU) = Array
```